source/Conventions.rst

```diff
@@ -25,7 +25,7 @@ Navigation
 User Interface & Configuration
 ==============================
 
-.. _ui-cli
+.. _ui-cli:
 
 Command Line Interface (CLI)
 ++++++++++++++++++++++++++++
```

Comment:
ln 85: are you now recommending that some of the P8 value_cal's be converted over to memo's?